scripts/Build.ps1

```diff
@@ -22,6 +22,12 @@ Param(
   [Alias("vs")]
   [System.String] $VersionSuffix = "dev",
   
+  [Parameter(Mandatory=$false)]
+  [System.String] $BuildVersionPrefix  = "14.0",
+
+  [Parameter(Mandatory=$false)]
+  [System.String] $BuildVersionSuffix = "99.99",
```

Comment:
How does it look like in real when provided by the CI ? Share an example.

Reply:
looks like 1926.01